### PR TITLE
Issue commenting bug

### LINKED ISSUE #43: Actions bot keeps commenting with task id on issue, even if it already exists
Need to build logic to not comment on issue with task id if the issue already exits

### PR DESCRIPTION
**Pull request by cheeragpatel**
Fixes #43 

Added function to check if totango id exists in metadata object.

### CODE REVIEW

File: src/totango.js [hunk 1]
```diff
@@ -417,7 +421,7 @@ async function closed({ issue }) {
   console.log('Issue was closed');
   // Check to see if the issue has a task associated with it
   // If it does, and the task is not already closed, close the task
-  let task_id = await issue_has_task_id({issue: issue});
+  let task_id = issue_has_totango_id({body});
```
Comment (collinmcneese) [inline src/totango.js:424]: Looks like this is trying to pass a `body` which is undefined within this function

File: src/totango.js [hunk 2]
```diff
@@ -377,6 +373,14 @@ async function labeled({ issue, label }) {
     // output the payload to the console so the user can see it
     console.log(`Touchpoint subject is: ${subject}`);
     console.log(`Touchpoint body is: ${body}`);
+    // check if touchpoint is already created for this issue (shouldn't be)
+    let check_touchpoint_id = issue_has_totango_id({body});
+    console.log(`Touchpoint id is: ${check_touchpoint_id}`)
```
[the remark this answers is not in the document]
Comment (cheeragpatel) [reply src/totango.js]: Made the suggested change

File: src/totango.js [hunk 3]
```diff
@@ -419,6 +419,7 @@ async function edited({ issue }){
 
 async function closed({ issue }) {
   console.log('Issue was closed');
+  let body = issue['body']
```
Comment (collinmcneese) [inline src/totango.js:422]:
```suggestion
  let body = issue['body'];
```

**Review by collinmcneese — APPROVED** [review]
🎉

**Pull request merged**